compiler/compiler/src/main/java/org/robovm/compiler/target/AbstractTarget.java

```diff
@@ -521,6 +521,22 @@ protected void stripArchive(Path path, File output) throws IOException {
         
         config.getLogger().info("Creating stripped archive file %s", output);
 
+        List<String> stripArchives = config.getStripArchives();
+        
+        /* Compile each stripArchives-Entry into a reusable Pattern. */
+        List<Pattern> stripArchivesPatterns = new ArrayList<>();
+        for (String stripArchivesEntry : stripArchives) {
+        	
+        		String javaRegex = stripArchivesEntry;
+        		
+        		javaRegex = javaRegex.replace(".", "\\.");
+        		javaRegex = javaRegex.replace("**", "{wildcard}");
+        		javaRegex = javaRegex.replace("*", "{wildcard}");
+        		javaRegex = javaRegex.replace("{wildcard}", ".*");
+        		
```

Comment:
I think it would be easier to just support java regexes, or is there a good reason not to? So we have kind of a mixture.